datalad/core/distributed/clone.py

```diff
@@ -585,15 +585,125 @@ def postclonecfg_ria(ds, props):
                    if s.get('annex-externaltype', None) == 'ora'
     ]
     if not ria_remotes:
-        lgr.debug("Found no RIA special remote")
+        # nothing autoenabled. Let's check origin's config for
+        # datalad.ora-remote.uuid as stored by create-sibling-ria and enable try
+        # enabling that one.
+        lgr.debug("Found no autoenabled ORA special remote. Trying to look it"
+                  "up in source config ...")
+
+        # First figure whether we cloned via SSH, HTTP or local path and then
+        # get that config file the same way:
+
+        config_content = None
+        scheme = props['source'].split(':')[0][4:]
+        if scheme == 'http':
+
+            # Note: Following outcommented code was proof how to check via HTTP.
+            #       However, for now we only want to annexenable origin in case
+            #       of HTTP, not the special remote. Hence it's uuid isn't
+            #       actually required. (see gh-4410)
+
+            # import requests
+            # try:
+            #     response = requests.get("{}{}config".format(
+            #         props['giturl'],
+            #         '/' if not props['giturl'].endswith('/') else '')
+            #     )
+            #     config_content = response.text
+            # except requests.RequestException as e:
+            #     lgr.debug("Failed to get config file from source:\n%s",
+            #               exc_str(e))
+
+            # config_content still None should lead to skip everything special
+            # remote related. Instead enable origin to deliver annex keys via
+            # HTTP:
+            lgr.debug("Trying to enable remote origin for git-annex")
+            ds.repo.enable_remote('origin')
+            ds.repo.fsck(remote='origin', fast=True)
```

Comment:
I assume the last line is needed, because git annex isn't aware of any availability at "origin", because the data was copied through the special remote. If that is correct, then I think we should not do this. The result would be that annex believe that two copies of the file exist, where there is only one -- thereby invalidating any minimum redundancy requirements.

Reply:
I'm struggling with the need for this as well. There are several possible cases: We could have store with no special remote at all, in which case `fsck` should be superfluous. So, even in case of HTTP we should prob. check origin's config and not call `fsck` if there's no special remote uuid to be found.
In case a special remote was indeed used for upload to the store, I think the `fsck` is needed, since otherwise there's no way to get any content via origin. But you're right - minimum redundancy would be torpedoed. I did not yet play with `sameas`. May be that would be a solution. So, if we decide to try to use origin we also declare it `sameas` the special remote.

Reply:
I think, if there nothing equivalent to `sameas` then the approach must be dropped.

Reply:
Played a bit with passing `sameas` and `uuid` parameters, but no success so far. So - will drop HTTP for now and do a proper implementation in ORA remote. But won't blow up this PR even more.